nservicebus/security/encryption.md

```diff
@@ -54,11 +55,11 @@ snippet:EncryptionServiceSimple
 
 #### Key ID
 
-Each key needs an unique key ID (`KeyIdentifier`). The key ID is communicated in the message header meta data and provides the receiving endpoint information on which key to use for decryption.
+Each key needs an unique key ID (`KeyIdentifier`). The key ID is communicated in the message header and allows the receiving endpoint to use the correct decryption key.
 
 > Error: It is required to set the Rijndael key ID.
 
-NOTE: If a key ID is not set then no encrypted messages can be send but received messages without a key ID header will be decrypted using all keys in the configuration.
+NOTE: If a key ID is not set then no messages with encrypted properties can be sent. When NServiceBus receives a message without a key ID header, it will attempt decryption using all keys in the configuration.
```

Comment:
@MarcinHoppe @ramonsmits what happens with the message then? is it moved to error queue? if so, what error message can one expect?